ProcessMaker/Http/Controllers/Admin/GroupController.php

```diff
@@ -29,4 +29,8 @@ public function edit(Group $group)
      return view('admin.groups.edit', ["group"=>$group]);
    }
 
+   public function show(Group $group) // show new process to UI
+   {
+       return view('admin.groups.show', ["group"=>$group]);  // from data item in index, once clicked, this page will show with ability to edit and destroy
```

Comment:
Use compact instead of passing in an array.

Reply:
complete